src/components/Home.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchData } from '../redux/covidData/covid';
+
+function Home() {
```

Comment:
- [ ] Based on the requirements, the Home component should have a filtering mechanism, say a search box, with which users can filter countries on the homepage.

  ![gim](https://user-images.githubusercontent.com/81468675/188887040-68d72b0c-c323-4d77-8189-a45d31d4764e.jpg)

- [ ] Based on the design being used, your Home page should have a banner section containing information about the countries listed on the home page

  ![fdd](https://user-images.githubusercontent.com/81468675/188887286-f01a2579-342e-46d1-9b5f-2af6a1658aa1.jpg)